src/Resources/translations/validators.de.xlf

```diff
@@ -1,79 +0,0 @@
-<?xml version="1.0"?>
-<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
-    <file source-language="en" datatype="plaintext" original="file.ext">
```

Comment:
Don't worry, I took care of this 👍 https://github.com/rollerworks/PasswordStrengthBundle/pull/80/files#diff-1542328f206d0842be7722bca01655dfR54